mutiny-core/src/ldkstorage.rs

```diff
@@ -973,18 +948,36 @@ mod test {
             scoring_params(),
         ));
 
+        // if regtest, we don't need to get the tip hash and can
+        // just use genesis, this also lets us use regtest in tests
+        let best_block = if network == Network::Regtest {
+            BestBlock::from_network(network)
+        } else {
+            let height_future = esplora
+                .get_height()
+                .map_err(|_| MutinyError::ChainAccessFailed);
+            let hash_future = esplora
+                .get_tip_hash()
+                .map_err(|_| MutinyError::ChainAccessFailed);
+            let (height, hash) = try_join!(height_future, hash_future)?;
+            BestBlock::new(hash, height)
+        };
+        let chain_params = ChainParameters {
+            network,
+            best_block,
+        };
```

Comment:
since this is a test shouldn't need to match on network and can just remove the else case here